src/test/java/com/iexec/core/replicate/ReplicateServiceTests.java

```diff
@@ -792,7 +792,7 @@ void shouldReturnFalseSinceTaskNotFound() {
 
         boolean isResultUploaded = replicatesService.isResultUploaded(CHAIN_TASK_ID);
         assertThat(isResultUploaded).isFalse();
-        verify(resultService, never()).isResultUploaded(CHAIN_TASK_ID);
+        verify(resultService, never()).isResultUploaded(iexecHubService.getTaskDescription(CHAIN_TASK_ID));
```

Comment:
In fact, a more interesting check can be done here:
```suggestion
        verifyNoInteractions(resultService);
```